robottelo/vm.py

```diff
@@ -56,13 +51,22 @@ def __init__(
             self, cpu=1, ram=512, distro=None, provisioning_server=None,
             image_dir=None, tag=None, hostname=None, domain=None,
             target_image=None):
+        distro_el6 = settings.distro.image_el6
+        distro_el7 = settings.distro.image_el7
+        base_images = [distro_el6, distro_el7]
         self.cpu = cpu
         self.ram = ram
-        self.distro = BASE_IMAGES[-1] if distro is None else distro
-        if self.distro not in BASE_IMAGES:
+        if distro == 'rhel6':
+            distro = distro_el6
+        if distro == 'rhel7':
+            distro = distro_el7
+        if distro is None:
+            distro = distro_el7
+        self.distro = distro
```

Comment:
Also, the whole idea of this change is to support custom base-images, so it perfectly fits in with the current approach.
```
[distro]
image_el6="rhel68"
image_el7="rhel73"
```
or 

```
[distro]
image_el6="rhel69"
image_el7="rhel72"
```


```
if distro == 'rhel6':
    distro == settings.distro.image_el6
if distro == 'rhel7':
    distro == settings.distro.image_el7
```